docs/modules/ROOT/pages/components.adoc

```diff
@@ -137,21 +138,23 @@ By adding the embed attribute, `is_initialized` becomes a contract entrypoint fo
 [TIP]
 ====
 Embeddable implementations, when available in this library's components, are segregated from the internal component implementation which makes it easier to safely expose.
-Components also separate standard implementations (`snake_case`) from `camelCase`.
-This trichotomy structures the API documentation design.
+Components also separate standard (granular) implementations from {mixin} implementations.
```

Comment:
```suggestion
Components also separate granular implementations from {mixin} implementations.
```
I think some granular impls implement standards, but that's the only relationship I see. We have some standards split into different impls, non-standard impls, etc